server.ts

```diff
@@ -11,10 +11,19 @@ import { env } from '#utils/env'
 import { Color } from '#utils/enums'
 import { errorHandler } from '#helper/errorHandler'
 import { HttpStatusCode } from 'axios'
+import { autoRetry } from '@grammyjs/auto-retry'
 import Model from '#config/database'
 
 const bot = new Bot<BotContext>(env.TOKEN)
 
+// plugins
+bot.api.config.use(
+  autoRetry({
```

Comment:
What it does

Reply:
It avoids the 429 (too many requests) error.

Reply:
telegramdan error kesa 1s kutib success keguncha qayta-qayta 3 marta urinib ko'radi